assets/css/general.css

```diff
@@ -26,6 +26,31 @@ h2 {
     z-index: 10000;
 }
 
+/* sass-lint:disable class-name-format*/
+.position-relative {
+    position: relative;
+}
+
+.d-none {
+    display: none;
```

Comment:
Don't forget that this requires JS changes! Try out the app and you'll see what I mean. It might be worth changing it back to inline if the transition gets lost. Not sure.

Reply:
sushain can u please tell Where exactly in Js are the changes needed specifically couldn't figure but runnig good in index.html